### PR TITLE
Fixed placeBlock example

### PR DESCRIPTION
**Pull request by IceTank**

### CODE REVIEW

**Comment by IceTank** [comment]
The command 'come' was not working.

**Comment by rom1504** [comment]
can you check this @Karang or @u9g ?

File: examples/placeBlock.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ bot.once('spawn', () => {
   bot.pathfinder.setMovements(defaultMove)
 
   bot.on('chat', async (username, message) => {
-    const target = bot.players[username]
+    const target = bot.players[username].entity
```
Comment (u9g) [inline examples/placeBlock.js:28]: if you just dont make this change, the you don't need to make the change below

**Review by u9g — CHANGES_REQUESTED** [review]
just one small thing, otherwise looks good

**Comment by Karang** [comment]
This is already merged ?

**Comment by u9g** [comment]
> This is already merged ?

oh I guess it was

**Pull request closed without merging**